doc/modules/classes.rst

```diff
@@ -1360,23 +1360,6 @@ Low-level methods
 Recently deprecated
 ===================
 
-To be removed in 0.19
```

Comment:
Don't worry about it. Yes, you have my +1. I can't be bothered going through the details a second time!